README.md

```diff
@@ -124,6 +124,39 @@ const mySteps: TourStep[] = [{
 
 You can also find a complete example [here](example/).
 
+## Built-in Helper Components
+
+You can take advantage of the already built-in customizable components. For example, our [TourBox] (https://stackbuilders.github.io/react-native-spotlight-tour/docs/build/#tourbox) component can be used as a tooltip container for each step.
```

Comment:
```suggestion
You can take advantage of the built-in customizable components. For example, our [TourBox](https://stackbuilders.github.io/react-native-spotlight-tour/docs/build/#tourbox) component can be used as a tooltip container for each step.
```

Reply:
@krarrobo1 we need to remove the whitespace in the `Link` syntax or the link render will break. Also, the `already` word seems a bit redundant, but not that important 🙂